gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/GremlinServer.java

```diff
@@ -108,54 +108,24 @@ public GremlinServer(final Settings settings) {
             workerGroup = new NioEventLoopGroup(settings.threadPoolWorker, threadFactoryWorker);
         }
 
-        serverGremlinExecutor = new ServerGremlinExecutor<>(settings, null, workerGroup, EventLoopGroup.class);
+        serverGremlinExecutor = new ServerGremlinExecutor(settings, null, workerGroup, null);
```

Comment:
i think you're right. I sorta wonder if the second parameter is of value. Gremlin Server is the only thing that really creates that ExecutorService.